src/Tutor.Infrastructure/EventConfiguration/EventSerializationConfiguration.cs

```diff
@@ -16,8 +16,8 @@ public static class EventSerializationConfiguration
         public static readonly IImmutableDictionary<Type, string> EventRelatedTypes = new Dictionary<Type, string>
         {
             { typeof(AssessmentItemAnswered), "AssessmentItemAnswered" },
-            { typeof(SoughtHints), "SoughtChallengeHints" },
-            { typeof(SoughtSolution), "SoughtChallengeSolution" },
+            { typeof(HintsRequested), "SoughtChallengeHints" },
+            { typeof(SolutionRequested), "SoughtChallengeSolution" },
```

Comment:
This is pretty touchy because of past events so I left their serialization discriminator the same for now, should I change it or not?

Reply:
Changed discriminators in 261bda392588fa08721371d66bc1149fd4a5cbce.